Projects/UOContent/Network/Packets/IncomingPlayerPackets.cs

```diff
@@ -365,37 +365,37 @@ public static void DisplayGumpResponse(NetState state, SpanReader reader)
 
         if (gump != null)
         {
-            var buttonExists = buttonId == 0; // 0 is always 'close'
-
-            if (!buttonExists)
-            {
-                foreach (var e in gump.Entries)
-                {
-                    if (e is GumpButton button && button.ButtonID == buttonId)
-                    {
-                        buttonExists = true;
-                        break;
-                    }
-
-                    if (e is GumpImageTileButton tileButton && tileButton.ButtonID == buttonId)
-                    {
-                        buttonExists = true;
-                        break;
-                    }
-                }
-            }
-
-            if (!buttonExists)
-            {
-                state.LogInfo("Invalid gump response, disconnecting...");
-                var exception = new InvalidGumpResponseException($"Button {buttonId} doesn't exist");
-                exception.SetStackTrace(new StackTrace());
-                NetState.TraceException(exception);
-                state.Mobile?.SendMessage("Invalid gump response.");
-
-                // state.Disconnect("Invalid gump response.");
-                return;
-            }
+            // var buttonExists = buttonId == 0; // 0 is always 'close'
+            //
+            // if (!buttonExists)
+            // {
+            //     foreach (var e in gump.Entries)
```

Comment:
Should we even bother recording buttons? I think the gump's OnResponse should properly handle invalid input anyways.